scripts/scil_compute_ihMT_maps.py

```diff
@@ -0,0 +1,378 @@
+# -*- coding: utf-8 -*-
+"""
+Computing ihMT and non-ihMT maps
+Make a more details description !
+"""
+# Verifier si import dans le bon ordre...
+import argparse
+from bids import BIDSLayout
+import math
+
+import numpy as np
+import nibabel as nib
+import scipy.ndimage
+
+from scilpy.io.utils import (add_overwrite_arg,
+                             assert_inputs_exist, assert_outputs_exist)
+
+# @Arnaud :ici j'ai un probleme avec le --resolution avec le type pour la ligne 305
+def _build_arg_parser():
+    p = argparse.ArgumentParser(description=__doc__,
+                                formatter_class=argparse.RawTextHelpFormatter)
+    p.add_argument('input',
+                   help='Path of the ihMT maps from bids folder')
+    p.add_argument('output',
+                   help='Path to output folder')
+    p.add_argument('subjid',
+                   help='Id of subject from BIDS folder')
+    p.add_argument('brainmask',
+                   help='Path to T1 binary brainmask.'
+                        'Generate by BET or other tools')
+    p.add_argument('--resolution', type=str,
+                   help="Use this option when 2 or most resolution is acquired"
+                        "name of ihMT sequences resolution as set in"
+                        "BIDSFile object, ex:['highres','lowres']."
+                        "Default is None.")
+    p.add_argument('--filtering', action='store_true',
+                   help='Gaussian filtering to remove Gibbs ringing,'
+                        'sigma set at 0.5. Not recommanded.'
+                        'Default is None')
+
+    add_overwrite_arg(p)
+
+    return p
+
+
+def set_acq_parameters(image):
+    TR = (layout.get_metadata(image)['RepetitionTime'])*1000
+    FlipAngle = ((layout.get_metadata(image)['FlipAngle'])*math.pi/180)
+    return TR, FlipAngle
+
+
+def merge_ihMT_array(images):
+    """
+    Function to concatenate n 3D-array matrix of the same size
+    along the 4th dimension.
+    Parameters
+    ----------
+    images       list of echo path. Must be nii image.
```

Comment:
check docstring in others scripts. You need the type of the variable and then a definition.
With python nii.gz is your friend.